tests/Unit/Http/TestCase.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace LaraCrafts\UrlShortener\Tests\Unit\Http;
+
+use Orchestra\Testbench\TestCase as BaseTestCase;
+
+abstract class TestCase extends BaseTestCase
```

Comment:
In my opinion this should not be called `TestCase`. It would be better to name it `HttpTestCase`.